packages/helm-charts/celo-fullnode/templates/health-check.config-map.yaml

```diff
@@ -9,4 +9,4 @@ metadata:
 data:
   health-check.sh: |-
     #!/bin/sh
-{{ include "common.node-health-check" . | indent 4 }}
+{{ include "common.node-health-check" (dict "maxpeers" .Values.geth.maxpeers "light_maxpeers" .Values.geth.light.maxpeers ) | indent 4 }}
```

Comment:
Hi @jcortejoso, I just tried a dry run install of celo-fullnode and I got this error. Is this a problem with the chart or did I make a mistake in my command?

```
$ helm install --debug --dry-run goodly-guppy ./celo-fullnode
install.go:178: [debug] Original chart version: ""
install.go:199: [debug] CHART PATH: /Users/piersy/projects/celo-monorepo/packages/helm-charts/celo-fullnode

Error: INSTALLATION FAILED: template: celo-fullnode/templates/health-check.config-map.yaml:12:3: executing "celo-fullnode/templates/health-check.config-map.yaml" at <include "common.node-health-check" (dict "maxpeers" .Values.geth.maxpeers "light_maxpeers" .Values.geth.light.maxpeers)>: error calling include: template: celo-fullnode/charts/common/templates/_helpers.tpl:379:40: executing "common.node-health-check" at <.Values.geth.readiness_probe_max_block_age_seconds>: nil pointer evaluating interface {}.geth
helm.go:88: [debug] template: celo-fullnode/templates/health-check.config-map.yaml:12:3: executing "celo-fullnode/templates/health-check.config-map.yaml" at <include "common.node-health-check" (dict "maxpeers" .Values.geth.maxpeers "light_maxpeers" .Values.geth.light.maxpeers)>: error calling include: template: celo-fullnode/charts/common/templates/_helpers.tpl:379:40: executing "common.node-health-check" at <.Values.geth.readiness_probe_max_block_age_seconds>: nil pointer evaluating interface {}.geth
INSTALLATION FAILED
main.newInstallCmd.func2
        helm.sh/helm/v3/cmd/helm/install.go:127
github.com/spf13/cobra.(*Command).execute
        github.com/spf13/cobra@v1.2.1/command.go:856
github.com/spf13/cobra.(*Command).ExecuteC
        github.com/spf13/cobra@v1.2.1/command.go:974
github.com/spf13/cobra.(*Command).Execute
        github.com/spf13/cobra@v1.2.1/command.go:902
main.main
        helm.sh/helm/v3/cmd/helm/helm.go:87
runtime.main
        runtime/proc.go:225
runtime.goexit
        runtime/asm_amd64.s:1371
(END)
```

Reply:
That is because you are not providing the required `values.yaml` file. Right now the helm chart are very coupled to celotool or the `values.yaml` file generated by celotool.